lambda/custom/state_handlers.js

```diff
@@ -393,7 +391,7 @@ function matchArticleToTitlesHelper(stateObj) {
         thisVar.response
           .speak(constants.strings.TITLE_SEARCH_MATCH_FAIL)
           .listen(constants.strings.TITLE_SEARCH_MATCH_FAIL);
-        thisVar.attributes['chosenArticle'] = '';
+        thisVar.attributes['chosenArticle'] = 'none';
```

Comment:
Isn't it better to assign null to this var instead of a value?

Reply:
It wants a value in there, otherwise it will throw an exception.  If you set attributes to null it throws exception.